ee/modules/110-istio/monitoring/prometheus-rules/revisions.yaml

```diff
@@ -81,7 +81,8 @@
       plk_protocol_version: "1"
       summary: There are Pods with istio sidecars, but without istio-injection configured
     expr: |
-      d8_istio_pod_revision{revision!="absent", desired_revision="absent"}
+      d8_istio_pod_revision{desired_revision="absent",revision!="absent"}
+      unless on (dataplane_pod) label_replace(kube_pod_labels{label_sidecar_istio_io_inject="true"}, "dataplane_pod", "$1", "pod", "(.+)")
```

Comment:
this logic should be in revisions_monitoring hook.

Reply:
fixed